packages/lib/src/vis/tiles/TiledLayer.tsx

```diff
@@ -20,38 +20,43 @@ function TiledLayer(props: Props) {
   const { baseLayerIndex, numLayers, tileSize } = api;
   const layerSize = api.layerSizes[layer];
 
-  const { visSize } = useAxisSystemContext();
+  const { abscissaConfig, ordinateConfig, visSize } = useAxisSystemContext();
   const { xVisibleDomain, yVisibleDomain } = useScaledVisibleDomains(layerSize);
 
-  // Transform visible domain to current level-of-detail array coordinates
   const tileOffsets = getTileOffsets(xVisibleDomain, yVisibleDomain, tileSize);
 
   // Sort tiles from closest to vis center to farthest away
   const center = new Vector2(sum(xVisibleDomain) / 2, sum(yVisibleDomain) / 2);
   sortTilesByDistanceTo(tileOffsets, tileSize, center);
 
   return (
-    // Tranforms to use level of details layer array coordinates
+    // Tranforms to handle axes flip and use level of details layer array coordinates
     <group
-      position={[-visSize.width / 2, -visSize.height / 2, layer / numLayers]}
-      scale={[
-        visSize.width / layerSize.width,
-        visSize.height / layerSize.height,
-        1,
-      ]}
+      scale={[abscissaConfig.flip ? -1 : 1, ordinateConfig.flip ? -1 : 1, 1]}
```

Comment:
Handle axes flip here.

Reply:
Is it necessary to have it separate from the `group` setting the `position` ?

Reply:
Not really, This can also be applied in `scale` and `position` in the other `group`